examples/jsm/node-editor/math/TrigonometryEditor.js

```diff
@@ -10,12 +10,19 @@ export class TrigonometryEditor extends BaseNode {
 
 		const node = new MathNode( MathNode.SIN, DEFAULT_VALUE );
 
-		super( 'Trigonometry', 1, node, 175 );
+		super( 'Angle / Trigonometry', 1, node, 220 );
```

Comment:
I wish I had separated Angle from Trigonometry. @mrdoob Some suggestion?

Reply:
Is this a node to get radians out of a degrees? If so, angle sounds better indeed.

Reply:
Yes, yes. A Node in Editor only for it.

Reply:
I think that sounds good yes 👍